rules/windows/process_creation/Possible Manipulation Of Tokens on a Windows computers remotely Detected via Impersonate.yml

```
@@ -0,0 +1,29 @@
+title: Possible Manipulation Of Tokens on a Windows computers remotely Detected via Impersonate
```

Since the detection is focused on the tool, not the behavior. It should be the Execution of the tool like this.

```suggestion
title: Impersonate Execution
```